### PR TITLE
Feature/add twemoji support

### PR DESCRIPTION
**Pull request by latiif**
### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Adds optional support for Twemoji across the website. (Not only in posts).

Emoji size adapts to the text.

It can be switched on or off from `config.toml`. Default is `true`.

Adds a post on Twemoji Support. 

### Issues Resolved

Add Twemoji support #344 

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [x] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files

### CODE REVIEW

**Comment by latiif** [comment]
## Screenshots of the suggested `twemoji-support.md` post:
### Without this change or with `useTwemoji = false`.
normal

### With this change, i.e. with `useTwemoji = true`.
changed

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <!DOCTYPE html>
+{{ if .Site.Params.useTwemoji }}
```
Comment (clement-pannetier) [inline layouts/_default/baseof.html:2]: I think it's not the right place to insert this code, I would suggest placing it into the `<head>` element.

File: exampleSite/config.toml [hunk 2]
```diff
@@ -46,6 +46,9 @@ disqusShortname = "yourdiscussshortname"
     # Series see also post count
     maxSeeAlsoItems = 5
 
+    # Use Twemoji
+    useTwemoji = true
```
Comment (clement-pannetier) [inline exampleSite/config.toml:50]: In the [gohugo docs](https://gohugo.io/getting-started/configuration/#all-configuration-settings), they use the `enableEmoji` config parameter to enable Emoji emoticons support for page content, so I suggest using `enableTwemoji` instead.

File: exampleSite/content/posts/twemoji-support.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
++++
+date = "2020-05-25"
+title = "Twemoji Support 🤩"
+description = "The post demonstrates Twemooji support"
```
Comment (clement-pannetier) [inline exampleSite/content/posts/twemoji-support.md:4]: You made a typo here, Twemoji, not Twemo**o**ji. 😉

Comment (latiif) [reply exampleSite/content/posts/twemoji-support.md:4]: Twem😳ji

**Comment by luizdepra** [comment]
Nice, thank you.

**Pull request merged**